pkg/sync/destination.go

```diff
@@ -123,3 +144,22 @@ func (i *ImageDestination) GetRepository() string {
 func (i *ImageDestination) GetTag() string {
 	return i.tag
 }
+
+func gcpTokenFromCreds(creds string) (string, time.Time, error) {
```

Comment:
Maybe this function is better to be put into pkg/sync/utils.go too.

Reply:
Moved it